src/components/app/MenuButtons/Permalink.js

```diff
@@ -35,8 +36,16 @@ export class MenuButtonsPermalink extends React.PureComponent<Props, State> {
   state = {
     fullUrl: '',
     shortUrl: '',
+    panelOpen: !!this.props.isNewlyPublished,
```

Comment:
Why do we need `panelOpen` state for this `Permalink` component? It shouldn't really need to know anything about its children's state. We can simplify this by only passing `open={isNewlyPublished}` to the child and compute this there.